NGitLab.Tests/UsersTests.cs

```diff
@@ -229,11 +229,19 @@ public async Task GetLastActivityDatesFromTheFuture()
     }
 
     [Test]
-    public async Task GetLastActivityDates_UsingNonAdminCredentials_ThrowsForbidden()
+    public async Task GetLastActivityDates_UsingNonAdminCredentials_ShowsOnlyUsersWithPublicProfile()
     {
         using var context = await GitLabTestContext.CreateAsync();
-        var exception = Assert.Throws<GitLabException>(() => context.Client.Users.GetLastActivityDatesAsync().ToArray());
-        Assert.That(exception.StatusCode, Is.EqualTo(HttpStatusCode.Forbidden));
+        var privateProfileUsers = context.AdminClient.Users.All.Where(u => u.PrivateProfile).Select(ppu => ppu.Username).ToList();
+        Assert.That(privateProfileUsers, Is.Not.Empty);
+
+        var lastActivityDates = context.Client.Users.GetLastActivityDatesAsync().ToList();
+        Assert.That(lastActivityDates, Is.Not.Empty);
+
+#pragma warning disable MA0002
+        var lastActivityDatesOfPrivateProfileUsers = lastActivityDates.Where(lad => privateProfileUsers.Contains(lad.Username)).ToList();
```

Comment:
Isn't it just adding StringComparison.Ordinal or StringComparer.Ordinal to the contains method to avoid the warning?

Reply:
You're completely right. I changed it ...